src/content/components/Triage/Triage.js

```diff
@@ -36,12 +36,21 @@ export class Triage extends React.PureComponent {
     });
     this.setState({loaded: true, bugs, prevIterationBugs, prevIteration: prevIteration.number});
   }
+  // Separate out bugs with needinfo, we don't want to triage them until the request is resolved
```

Comment:
You need to add `flags` to line 27 for the `include_fields` columns in order for flags to be included